concepts/40 Angular Components/10 Getting Started/02 Create a DevExtreme Application/30 Configure the Navigation Menu/20 Hide the Menu in the Closed State.md

```diff
@@ -0,0 +1,11 @@
+In the closed state, the navigation menu is still partially visible because it displays item icons. If the items lack icons, it is better to entirely hide the menu. To do this, open the `SideNavOuterToolbarComponent` or `SideNavInnerToolbarComponent` (depending on the used [layout](/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/#Layouts)) and change the `updateDrawer()` function as follows:
```

Comment:
"still partially" Adverb adverb?

Reply:
If something is visible, it is visible?

Reply:
"it is better to" Why is it better to do this? This sounds more like an opinion than a fact.

Reply:
```suggestion
In the closed state, the navigation menu is visible because it displays item icons. If the items do not have icons, you can hide the menu. To do this, open the `SideNavOuterToolbarComponent` or `SideNavInnerToolbarComponent` (depending on the [layout](/Documentation/Guide/Angular_Components/Getting_Started/Create_a_DevExtreme_Application/#Layouts)) and change the `updateDrawer()` function as follows:
```